server/resources/style-light.json

```diff
@@ -1,1999 +1,3180 @@
+        "text-halo-color": "#e0e0e0",
+        "text-halo-width": 2
       }
     },
     {
-      "id": "continent",
+      "id": "places_country",
       "type": "symbol",
-      "source": "immich-map",
-      "source-layer": "place",
-      "maxzoom": 1,
-      "filter": ["all", ["==", "class", "continent"]],
+      "source": "protomaps",
+      "source-layer": "places",
+      "filter": [
+        "==",
+        "pmap:kind",
+        "country"
+      ],
       "layout": {
-        "text-field": "{name_en}",
-        "text-font": ["Open Sans Italic"],
-        "text-size": 13,
-        "text-transform": "uppercase",
-        "text-justify": "center"
+        "symbol-sort-key": [
+          "get",
+          "pmap:min_zoom"
+        ],
+        "text-field": "{name}",
+        "text-font": [
+          "Noto Sans Medium"
+        ],
+        "text-size": [
+          "interpolate",
+          [
+            "linear"
+          ],
+          [
+            "zoom"
+          ],
+          2,
+          [
+            "case",
+            [
+              "<",
+              [
+                "get",
+                "pmap:population_rank"
+              ],
+              10
+            ],
+            8,
+            [
+              ">=",
+              [
+                "get",
+                "pmap:population_rank"
+              ],
+              10
+            ],
+            12,
+            0
+          ],
+          6,
+          [
+            "case",
+            [
+              "<",
+              [
+                "get",
+                "pmap:population_rank"
+              ],
+              8
+            ],
+            10,
+            [
+              ">=",
+              [
+                "get",
+                "pmap:population_rank"
+              ],
+              8
+            ],
+            18,
+            0
+          ],
+          8,
+          [
+            "case",
+            [
+              "<",
+              [
+                "get",
+                "pmap:population_rank"
+              ],
+              7
+            ],
+            11,
+            [
+              ">=",
+              [
+                "get",
+                "pmap:population_rank"
+              ],
+              7
+            ],
+            20,
+            0
+          ]
+        ],
+        "icon-padding": [
+          "interpolate",
+          [
+            "linear"
+          ],
+          [
+            "zoom"
+          ],
+          0,
+          2,
+          14,
+          2,
+          16,
+          20,
+          17,
+          2,
+          22,
+          2
+        ],
+        "text-transform": "uppercase"
       },
       "paint": {
-        "text-color": "#633",
-        "text-halo-color": "rgba(255,255,255,0.7)",
-        "text-halo-width": 1
+        "text-color": "#a3a3a3"
       }
     }
   ],
-  "id": "immich-map-light"
+  "sprite": "https://protomaps.github.io/basemaps-assets/sprites/v3/dark",
```

Comment:
Does it matter that the light style uses dark sprites?

Reply:
Ah yes this should be changed, good spot